### PR TITLE
[improvement](compaction)(tablet repair) Add missing rowsets in compaction status url and support force dropping redundant replica

### PR DESCRIPTION
**Pull request by morningman**
## Proposed changes

1. Add missing rowsets in compaction status url
2. Add a new config `force_drop_redundant_replica` to force drop redundant replicas.
3. Fix FE ut

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)
- [ ] Optimization. Including functional usability improvements and performance improvements.
- [ ] Dependency. Such as changes related to third-party components.
- [ ] Other.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/zh-CN/administrator-guide/http-actions/compaction-action.md [hunk 1]
```diff
@@ -84,6 +84,7 @@ curl -X GET http://be_host:webserver_port/api/compaction/show?tablet_id=xxxx\&sc
         "[50-50] 0 DELETE NONOVERLAPPING 574.00 B",
         "[51-51] 5 DATA OVERLAPPING 574.00 B"
     ],
+    "missing_rowsets": [],
```
Comment (EmmyMiao87) [inline docs/zh-CN/administrator-guide/http-actions/compaction-action.md:87]: 如果 compaction 的时候发现有 missing 的row set 那数据不就直接错误了嘛？
那还有必要知道丢了哪些版本嘛？直接标记tablet error？

File: fe/fe-core/src/main/java/org/apache/doris/common/Config.java [hunk 2]
```diff
@@ -1544,4 +1544,15 @@
      */
     @ConfField(mutable = true, masterOnly = true)
     public static boolean disable_tablet_scheduler = false;
+
+	/*
```
Comment (EmmyMiao87) [inline fe/fe-core/src/main/java/org/apache/doris/common/Config.java:1548]: code format

File: fe/fe-core/src/main/java/org/apache/doris/common/Config.java [hunk 3]
```diff
@@ -1544,4 +1544,15 @@
...
+     * When doing clone or repair tablet task, there may be replica is REDUNDANT state, which
+     * should be dropped later. But there are be loading task on these replicas, so the default strategy
+     * is to wait until the loading task finished before dropping them.
+     * But the default strategy may takes very long time to handle these redundant replicas.
+     * So we can set this config to true to not wait any loading task.
+     * Set this config to true may cause loading task failed, but will speed up the process of tablet balance and repair.
+     */
+    @ConfField(mutable = true, masterOnly = true)
+    public static boolean force_drop_redundant_replica = false;
```
Comment (EmmyMiao87) [inline fe/fe-core/src/main/java/org/apache/doris/common/Config.java:1557]:
```suggestion
    public static boolean enable_force_drop_redundant_replica = false;
```

File: docs/zh-CN/administrator-guide/http-actions/compaction-action.md (hunk 1 above)
Comment (morningman) [reply docs/zh-CN/administrator-guide/http-actions/compaction-action.md:87]: This is not just for compaction, this is also for user to see the details of rowsets in a tablet.

File: fe/fe-core/src/main/java/org/apache/doris/common/Config.java (hunk 3 above)
Comment (morningman) [reply fe/fe-core/src/main/java/org/apache/doris/common/Config.java:1557]: done

**Review by EmmyMiao87 — DISMISSED** [review]
LGTM

**Review by EmmyMiao87 — APPROVED** [review]
LGTM

**Pull request merged**